include/boost/http_proto/serializer.hpp

```diff
@@ -354,6 +468,8 @@ struct serializer::stream
 
 //---------------------------------------------------------
 
+/** A ConstBufferSequence representing the output
```

Comment:
Talking about input and output areas is great, but it needs to be explained. For example in the `serializer` class declaration javadoc you need to introduce these terms. Here is one way to do it:

"The serializer refers to notional buffer regions called the 'input area' and the 'output area'. During operation the caller is responsible for providing data in the input area either directly such as when using caller-provided buffer sequences via `stream`, or indirectly such as when using a Body type such as `file_body`. After input is provided, serialized data becomes available in the serializer's output area in the form of a constant buffer sequence. The caller alternates between filling the input area and consuming output area until all the input has been provided and all the output has been consumed, or an error occurs."